src/app/shared/support/support.scss

```diff
@@ -0,0 +1,66 @@
+.docs-help-support {
+  width: 75%;
+  margin: auto;
+  padding-bottom: 40px;
+
+  ul {
```

Comment:
This is more of a style thing than anything else, but we prefer to target the lowest specificity possible:
https://github.com/angular/components/blob/master/CODING_STANDARDS.md#use-lowest-specificity-possible

In particular, you don't need to nest at every level; everything could be nested directly under `.docs-help-support`. That said, my personal preference is to use CSS classes instead of native element selectors because it gives the flexibility to change the elements later without throwing off the styles.